### PR TITLE
Blacklist2idea hotfix - Description/Note issue

### PR DESCRIPTION
**Pull request by cejkato2**
Description and Note were reworked for new blacklist2idea.
Description contains only blacklist name (which will probably not change very often).
Note is text representation which contain very flexible information that changes from one event to another.

### CODE REVIEW

File: report2idea/blacklist/blacklist2idea.py [hunk 1]
```diff
@@ -105,13 +103,11 @@ def set_common_ip_fields(self):
...
     def set_common_url_fields(self):
-        self.idea["Note"] = IdeaTemplate.note.format(self.rec["type"].upper(),
-                                                     self.rec["source_url"],
-                                                     self.bl["name"])
-
-        self.idea["Description"] = IdeaTemplate.description_url.format(self.rec["source_url"],
-                                                                       self.bl["name"],
-                                                                       ip_list2description(self.rec["targets"]))
+        self.idea["Description"] = "Suspicious communication with domain name listed on {0} blacklist".format(self.bl["name"])
+        self.idea["Note"] = IdeaTemplate.url_note.format(self.rec["source_url"],
```
Comment (sustefil) [inline report2idea/blacklist/blacklist2idea.py]: bad variable name here: url_note -> note_url

File: report2idea/blacklist/blacklist2idea.py [hunk 2]
```diff
@@ -105,13 +103,11 @@ def set_common_ip_fields(self):
...
 
     def set_common_url_fields(self):
-        self.idea["Note"] = IdeaTemplate.note.format(self.rec["type"].upper(),
-                                                     self.rec["source_url"],
-                                                     self.bl["name"])
-
-        self.idea["Description"] = IdeaTemplate.description_url.format(self.rec["source_url"],
-                                                                       self.bl["name"],
-                                                                       ip_list2description(self.rec["targets"]))
+        self.idea["Description"] = "Suspicious communication with domain name listed on {0} blacklist".format(self.bl["name"])
```
Comment (sustefil) [inline report2idea/blacklist/blacklist2idea.py]: maybe replace "domain name" with "domain name/URL", since both of these can occur as the "source_url"

**Comment by vaclavbartos** [comment]
I agree with what @sustefil proposes, otherwise it's OK.

File: report2idea/blacklist/blacklist2idea.py (hunk 1 above)
Comment (cejkato2) [reply report2idea/blacklist/blacklist2idea.py]: This was fixed

File: report2idea/blacklist/blacklist2idea.py (hunk 2 above)
Comment (cejkato2) [reply report2idea/blacklist/blacklist2idea.py]: Fixed in 273bc83

**Pull request merged**